decidim-admin/lib/decidim/admin/test/manage_attachments_examples.rb

```diff
@@ -1,102 +1,101 @@
-
-      attach_file :attachment_file, Decidim::Dev.asset("Exampledocument.pdf")
-      find("*[type=submit]").click
-    end
+    it "can view an attachment details" do
+      within "#attachments table" do
+        click_link translated(attachment.title, locale: :en)
+      end
 
-    within ".callout-wrapper" do
-      expect(page).to have_content("successfully")
+      expect(page).to have_selector("input#attachment_title_en[value='#{translated(attachment.title, locale: :en)}']")
+      expect(page).to have_selector("input#attachment_description_en[value='#{translated(attachment.description, locale: :en)}']")
+      expect(page).to have_css("img[src~='#{attachment.url}']")
     end
 
-    within "#attachments table" do
-      expect(page).to have_link("Very Important Document")
-    end
-  end
+    it "can add attachments to a process" do
+      find(".card-title a.new").click
+
+      within ".new_participatory_process_attachment" do
+        fill_in_i18n(
+          :attachment_title,
+          "#attachment-title-tabs",
+          en: "Very Important Document",
+          es: "Documento Muy Importante",
+          ca: "Document Molt Important"
+        )
+
+        fill_in_i18n(
+          :attachment_description,
+          "#attachment-description-tabs",
+          en: "This document contains important information",
+          es: "Este documento contiene información importante",
+          ca: "Aquest document conté informació important"
+        )
+
+        attach_file :attachment_file, Decidim::Dev.asset("Exampledocument.pdf")
+        find("*[type=submit]").click
+      end
 
-  it "can delete an attachment from a process" do
-    within find("tr", text: stripped(translated(attachment.title))) do
-      page.find("a.action-icon--remove").click
-    end
+      within ".callout-wrapper" do
+        expect(page).to have_content("successfully")
+      end
 
-    within ".callout-wrapper" do
-      expect(page).to have_content("successfully")
+      within "#attachments table" do
+        expect(page).to have_link("Very Important Document")
+      end
     end
 
-    expect(page).not_to have_content(translated(attachment.title, locale: :en))
-  end
-
-  it "can update an attachment" do
-    within "#attachments" do
+    it "can delete an attachment from a process" do
       within find("tr", text: stripped(translated(attachment.title))) do
-        page.find("a.action-icon--edit").click
+        page.find("a.action-icon--remove").click
       end
-    end
 
-    within ".edit_participatory_process_attachment" do
-      fill_in_i18n(
-        :attachment_title,
-        "#attachment-title-tabs",
-        en: "This is a nice photo",
-        es: "Una foto muy guay",
-        ca: "Aquesta foto és ben xula"
-      )
+      within ".callout-wrapper" do
+        expect(page).to have_content("successfully")
+      end
 
-      find("*[type=submit]").click
+      expect(page).not_to have_content(translated(attachment.title, locale: :en))
```

Comment:
Note that this was only an identation change from wrapping the specs in a `context` block, to be consistent with the rest of the file. Check the diff with whitespace ignored: https://github.com/decidim/decidim/pull/1723/files?w=1.

In any case, yes, `have_no_content` is better. I can fix it since I'm at it (we should probably introduce `rubocop-rspec` to enforce this kind of things).